tests/autoyast/installation.pm

```diff
@@ -131,7 +131,8 @@ sub run {
 
     test_ayp_url;
     my $test_data = get_test_suite_data();
-    my @needles = qw(bios-boot nonexisting-package reboot-after-installation linuxrc-install-fail scc-invalid-url warning-pop-up autoyast-boot package-notification nvidia-validation-failed);
+    my @needles = qw(bios-boot nonexisting-package reboot-after-installation linuxrc-install-fail scc-invalid-url warning-pop-up autoyast-boot package-notification nvidia-validation-failed import-untrusted-gpg-key);
```

Comment:
Did you add new 'import-untrusted-gpg-key' needle?

If you've added it just locally, could you please also make a PR to needles repo and paste the link to it.

Reply:
no this exists already